lib/src/ui/pager_indicator.dart

```diff
@@ -16,8 +17,14 @@ class PagerIndicator extends StatelessWidget {
   Widget build(BuildContext context) {
     // extracting page bubble information from page view model
     final bubbles = <PageBubble>[];
+    final numOfPages = viewModel.pages.length;
 
-    for (var i = 0; i < viewModel.pages.length; i++) {
+    // calculates the width of the bubble to avoid the overflowing render issue #96
+    final bubbleWidth = BUBBLE_WIDTH * numOfPages > context.screenWidth
+        ? (context.screenWidth / numOfPages)
+        : BUBBLE_WIDTH;
+
+    for (var i = 0; i < numOfPages; i++) {
       final page = viewModel.pages[i];
 
```

Comment:
moved the calculation here so the bubble width gets calculated once and not for each bubble, since all bubbles have the same width